src/Microsoft.Sbom.Parsers.Spdx30SbomParser/ComplianceStandard/InvalidElementInfo.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Sbom.Parsers.Spdx30SbomParser.ComplianceStandard.Enums;
+using Microsoft.Sbom.Parsers.Spdx30SbomParser.ComplianceStandard.Interfaces;
+
+namespace Microsoft.Sbom.Parsers.Spdx30SbomParser.ComplianceStandard;
+
+public class InvalidElementInfo
```

Comment:
As mentioned elsewhere, this probably should not be version-specific

Reply:
Handled in https://github.com/microsoft/sbom-tool/pull/993